solvency2sf/default.py

```diff
@@ -41,19 +79,35 @@ def scr_def(type1, type2):
     for j in range(len(type1g)):
         v_intra += 1.5 * dp[j] * (1 - dp[j]) / (2.5 - dp[j]) * lgd2[j]
 
+    # Article 201
     v_type1 = v_intra + v_inter
+    # Article 200.4
     sd_type1 = (v_type1)**0.5
     total_lgd = type1g.lgd.sum()
+
+    # Article 200
     sd_to_lgd = sd_type1 / total_lgd
+    # Article 200.1
     if sd_to_lgd <= 0.07:
         scr_def_t1 = 3 * sd_type1
+    # Article 200.2
     elif 0.07 < sd_to_lgd <= 0.2:
         scr_def_t1 = 5 * sd_type1
```

Comment:
@Bilal-KLH could you change this so the variable doesn't have the same name as the function?
e.g.
`t1 = 3 * sd_type1
...
return t1`